server/container/guice/cassandra-guice/src/test/java/org/apache/james/CassandraJamesServerFixture.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.james;
+
+import org.apache.james.jmap.draft.JmapJamesServerContract;
+import org.apache.james.modules.TestJMAPServerModule;
+
+public class CassandraJamesServerFixture {
```

Comment:
a fixture is usually a way to share data between tests
BTW, it's not easy to find a name to this class and I'm still convinced it's because it should not exist at all: it would hide the fact we are not able to build a server easily enough. Why don't you fix the root cause instead if you don't like copy/pasting these few lines around?

Reply:
I have reused the style in place for the others products.
Maybe we should create a ticket to refactor this globally ?

Reply:
Sorry, I didn't caught this one earlier. That being said, if we agree that it's not something we want, we can start by not using that style anymore.